README.md

```diff
@@ -1 +1,200 @@
-# Hello-World
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Set up Hello Microverse Project <a name="about-project"></a>
+
+> This is my project to practics this tools
+
+<u><b>Set up Hello Microverse Project</b></u> is a Simple excersie project to master all of our tools and best practices that we learned about in previous steps to use them in all Microverse projects.
+
+## 🛠 Built With <a name="built-with"></a>
+___
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Style</summary>
+  <ul>
+    <li><a href="">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Linters</summary>
+  <ul>
+    <li><a href="">Linters</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+____
+> Describe between 1-3 key features of the application.
+
+- **[Added index.html file]**
+- **[Added style.css file]**
+- **[Added Linter]**
+
```

Comment:
Please refactor this and add valid features of the project you're working on. Example 
Key Features
```
Use linter
Use Professional documentation
Use HTML and CSS
```